drivers/spi/spi_silabs_eusart.c

```diff
@@ -152,6 +217,330 @@ static int spi_silabs_eusart_configure(const struct device *dev, const struct sp
 	return 0;
 }
 
+static inline void spi_silabs_eusart_pm_policy_get(const struct device *dev)
+{
+	pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	pm_policy_state_lock_get(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
+	pm_device_runtime_get(dev);
+}
+
+static inline void spi_silabs_eusart_pm_policy_put(const struct device *dev)
+{
+	pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	pm_policy_state_lock_put(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
+	pm_device_runtime_put(dev);
+}
+
+#if defined(CONFIG_PM_DEVICE)
```

Comment:
See https://github.com/zephyrproject-rtos/zephyr/pull/85386 -- the PM action should be unconditionally used together with `pm_device_driver_init()` for all flavors of PM to work correctly:
* CONFIG_PM
* CONFIG_PM_DEVICE + CONFIG_PM_DEVICE_SYSTEM_MANAGED
* CONFIG_PM_DEVICE + CONFIG_PM_DEVICE_RUNTIME